client/devdocs/design/index.jsx

```diff
@@ -50,14 +53,22 @@ import ExternalLink from 'components/external-link/docs/example';
 import FeatureGate from 'components/feature-example/docs/example';
 import FilePickers from 'components/file-picker/docs/example';
 import Collection from 'devdocs/design/search-collection';
+import fetchUsageStats from 'state/usage-stats/actions';
 
-export default React.createClass( {
+let DesignAssets = React.createClass( {
 	displayName: 'DesignAssets',
 
 	getInitialState() {
 		return { filter: '' };
 	},
 
+	componentWillMount() {
+		if ( config.isEnabled( 'devdocs/usage-stats' ) ) {
+			const { dispatchFetchUsageStats } = this.props;
+			dispatchFetchUsageStats();
```

Comment:
If we fetch the stats in the controller then I wonder if we need redux at all.
Also if we move the fetch to the controller then would you wait until the request is complete before rendering the page or would you re-render when we get the stats?

By the way I took a look at the search page `client/devdocs/main.jsx` and the fetch is done in the component with [DocService.search](https://github.com/Automattic/wp-calypso/blob/master/client/devdocs/main.jsx#L92).